sources/missing.json

```diff
@@ -3,8 +3,10 @@
 	"fullname": "Missing",
 	"name": "missing",
 	"prefix": "xx",
-    	"key":"",
+	"key": "",
 	"url": "",
-	"license": "",
-        "description": "Missing - as in a placeholder for WOF records without a geometry"
-}
+	"license": "N/A",
+	"license_type": "",
+	"license_text": "",
+	"description": "Missing - as in a placeholder for WOF records without a geometry."
```

Comment:
File a followup issue to clarify what `missing` and `unknown` are in these situations. Are they prefixes, are they values, or properties? An example would  help.